AMD/fx.md

```diff
@@ -347,6 +356,9 @@ System Integrity Protection bitmask
 
 csr-active-config by default is set to `00000000` which enables System Integrity Protection. You can choose a number of different values but overall we recommend keeping this enabled for best security practices. More info can be found in our troubleshooting page: [Disabling SIP](../troubleshooting/troubleshooting.md#disabling-sip)
 
+* **run-efi-updater**: `No`
+  * this is used to prevent Apple's firmware update packages from installing, this is very important especially with PC hardware as these firmware updates will crash on updates.
```

Comment:
```suggestion
  * this is used to prevent Apple's firmware update packages from installing and breaking boot order; this is important as these firmware updates (meant for Macs) will not work.
```
I *think* this is right not sure